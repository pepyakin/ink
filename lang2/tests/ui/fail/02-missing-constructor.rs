#![feature(proc_macro_hygiene)]

use ink_lang as ink;

#[ink::contract(
    env = DefaultSrmlTypes,
    version = [0, 1, 0],
)]
mod noop {
    #[ink(storage)]
    struct Noop {}
}

fn main() {}
