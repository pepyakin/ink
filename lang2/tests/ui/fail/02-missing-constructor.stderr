error: ink! contracts require at least one constructor function declared with `#[ink(constructor)]`
  --> $DIR/02-missing-constructor.rs:9:1
   |
9  | / mod noop {
10 | |     #[ink(storage)]
11 | |     struct Noop {}
12 | | }
   | |_^
